packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -599,7 +606,7 @@ class C4DMastheadL1 extends StableSelectorMixin(LitElement) {
       [`${prefix}--masthead__l1-dropdown-column-wide`]: hasWide && isWide,
     });
     return hasWide
-      ? html`<div class=${classes}>${renderedSections}</div>`
+      ? html`<div class=${classes} part="column-wide">${renderedSections}</div>`
```

Comment:
Let’s call this `dropdown-column` generically, and then conditionally add a wide or narrow modifier, eg.
```suggestion
      ? html`<div class=${classes} part="dropdown-column${hasWide && !isWide ? ' dropdown-column--narrow' :  hasWide && isWide ? ' dropdown-column--wide' : ''}">${renderedSections}</div>`
```
Also update docs accordingly.